lib/internal/Magento/Framework/File/Uploader.php

```diff
@@ -220,7 +220,16 @@ public function save($destinationFolder, $newFileName = null)
 
         $destinationFile = self::_addDirSeparator($destinationFile) . $fileName;
 
-        $this->_result = $this->_moveFile($this->_file['tmp_name'], $destinationFile);
+        try {
+            $this->_result = $this->_moveFile($this->_file['tmp_name'], $destinationFile);
+        } catch (\Exception $e) {
+            // if the file exists and we had an exception continue anyway
+            if (file_exists($destinationFile)) {
```

Comment:
In case moving of the file really fails, this condition may be true, if destination file with the same name already exists. In this case, the issue will be hidden and it will be difficult to discover it.

I'd recommend to avoid total hiding of exceptions, especially, if we catch general exception. In case, we 100% sure what kind of exception we catch, it might be ignored if required by the business logic, but it would make sense at least log it.